pg_bm25/src/index_access/utils.rs

```diff
@@ -368,12 +391,14 @@ mod tests {
 
     #[pg_test]
     fn test_lookup_index_tupdesc() {
+        crate::setup_background_workers();
```

Comment:
I wonder if this is something you can set up in the `setup()` function in `lib.rs`, have you tried?